organizations/user.go

```diff
@@ -507,3 +507,42 @@ func mustObjectID(s string) primitive.ObjectID {
 	}
 	return id
 }
+// Activate single member in an organization
+func ReactivateMember(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	member_collection, org_collection := "members", "organizations"
+	vars := mux.Vars(r)
+	orgId, memberId := vars["id"], vars["mem_id"]
+
+	pOrgId, err := primitive.ObjectIDFromHex(orgId)
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": pOrgId})
+	if orgDoc == nil {
+		fmt.Printf("org with id %s doesn't exist!", orgId)
+		utils.GetError(errors.New("operation failed"), http.StatusBadRequest, w)
+		return
+	}
+
+	ActivatedMember := bson.M{"deleted": false, "deleted_at": "active"}
+	res, err := utils.UpdateOneMongoDbDoc(member_collection, memberId, ActivatedMember)
```

Comment:
DeletedAt   time.Time              `json:"deleted_at" bson:"deleted_at"`

Doing "deleted_at": "active" is wrong. the field is a time field and "active" is a string. 
Do this instead bson.M{"deleted": false, "deleted_at": time.Time{}} 
This sets a default time which they consider as zero time because timt.Time fields are never empty, they hold a default value of 0001-01-01 00:00:00 +0000 UTC when the fields are not set. Use this and test that it works fine.